manageiq-providers-redfish.gemspec

```diff
@@ -14,6 +14,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,lib}/**/*"]
 
   s.add_runtime_dependency "redfish_client", "~> 0.1"
+  s.add_runtime_dependency "server_sent_events", "~> 0.1"
```

Comment:
Is this a redfish specific gem or something general?  I would have thought this would be part of the `redfish_client` or at least something like `redfish-events`?

Reply:
This is a HTML standard: https://html.spec.whatwg.org/multipage/server-sent-events.html

But as I already said, I will abstract away some of the details in the `redfish_client` gem in order to present more standard interface to the ManageIQ.